great-idea-website/css/index.css

```
@@ -63,4 +63,134 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+body {
+	margin: 0px auto 0px auto;
+	width: 880px;
+	font-size: 16px;
+}
```

```suggestion
body {
	margin: 0px auto 0px auto;
	width: 880px;
	font-size: 16px;
}
``` soo where you are doing margin: 0px auto 0px auto; you could use shorthand like this margin: 0 auto; does the same thing